ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayload.java

```diff
@@ -252,4 +255,17 @@ JsonRpcResponse respondWithInvalid(
         requestId,
         new EnginePayloadStatusResult(INVALID, latestValidHash, Optional.of(validationError)));
   }
+
+  private void logImportedBlockInfo(final Block block, final double timeInS) {
+    LOG.info(
+        String.format(
+            "Imported #%,d / %d tx / %,d (%01.1f%%) gas / base fee %s / (%s) in %01.3fs.",
```

Comment:
Nit: could we put basefee between tx and gas?  We are dropping ommers and we could just re-use the position for basefee.

Reply:
Question: is dropping peer count intentional?  Does the ELs no longer gossip?  If there is still gossip I think we want peer counts.

Reply:
1. Moved the base fee where ommes where
2. I was thinking on logging peers count in another way, but not yet sure how, so it make sense to readd them, since there is still gossip